fe/fe-core/src/main/java/org/apache/doris/qe/ShowExecutor.java

```diff
@@ -1834,7 +1837,56 @@ private void handleShowQueryProfile() throws AnalysisException {
 
         resultSet = new ShowResultSet(showStmt.getMetaData(), rows);
     }
-
+    
+    private void handleShowCreateRoutineLoad() throws AnalysisException {
+        ShowCreateRoutineLoadStmt showCreateRoutineLoadStmt = (ShowCreateRoutineLoadStmt) stmt;
+        List<List<String>> rows = Lists.newArrayList();
+        String dbName = showCreateRoutineLoadStmt.getDb();
+        String labelName = showCreateRoutineLoadStmt.getLabel();
+        // if include history return all create load
+        if (showCreateRoutineLoadStmt.isIncludeHistory()) {
+            List<RoutineLoadJob> routineLoadJobList = new ArrayList<>();
+            try {
+                routineLoadJobList = Catalog.getCurrentCatalog().getRoutineLoadManager().getJob(dbName, labelName, true);
```

Comment:
The 'getJob' function will return null when there is no routine load has been found.